packages/website/src/docs/configuration/tracing.md

```diff
@@ -0,0 +1,66 @@
+---
+id: tracing
+title: Tracing
+---
+
+# Tracing
+
+By default the Cadl Compiler will build without any debug information. The standard output will be minimal and limited to any `warning` or `error` diagnostics emitted during compilation.
+
+Some additional information is however being collected and can be revealed using the `--trace` cli flag.
+
+```bash
+cadl compile . --trace import-resolution
+```
+
+You can use the `--trace` option multiple times if there is multiple areas that should be logged from.
+
+```bash
+cadl compile . --trace import-resolution  --trace projection
+```
```

Comment:
I take it `--trace *` is the way to get absolutely all tracing?

Reply:
indeed

Reply:
i'll add a doc for that